### ISSUE TITLE
Last Slider Demo broken in IE11 and Edge

### ISSUE THREAD

**Issue by fabsenet** [angular/material#9492]
I am taking a look at the last slider demo on https://material.angularjs.org/latest/demo/slider (Volume/Bass/Master/...) and it just does not look right. I made a screenshot comparing the different browsers, the error should be obvious:
![angular_material_slider_demo_broken](https://cloud.githubusercontent.com/assets/2065705/18200173/e031f19a-7103-11e6-99a2-929b082603af.jpg)

**Actual Behavior**:
 -  `What is the issue? *` Display error
 -  `What is the expected behavior?` All browsers should render identical

**CodePen** (or steps to reproduce the issue): * open https://material.angularjs.org/latest/demo/slider on different browsers
 -  `CodePen Demo which shows your issue:`
 -  `Details:`

**Angular Versions**: * latest?
 -  `Angular Version:`
 -  `Angular Material Version:`

**Additional Information**:
 -  `Browser Type: *` Chrome, IE, Edge
 -  `Browser Version: *` Chrome 52, IE11, Microsoft Edge 38.14393.0.0
 -  `OS: *` Windows 10 Anniversary Update
 -  `Stack Traces:` -

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by fabsenet** [comment]
I investigated a little further.

The actual demo file appears to be fine: https://github.com/angular/material/blob/master/src/components/slider/demoVertical/index.html

When I click the integrated "view source" button on demo page, the source is also fine.

When I click the "open in codepen" button, the html code in the codepen is messed up. The layouting is wrong: For example the text "volume" is not inside the md-slider-container.

**This looks to me like an issue in your website deployment not using the current demo file for all usages.**

**Comment by Splaktar** [comment]
Indeed, it appears that IE11 is actually changing the DOM and moving the `h5` out of the `md-slider-container`. Wild!
![slider-labelsoutsidecontainer-ie11](https://user-images.githubusercontent.com/3506071/44811095-b7690e80-aba1-11e8-9a48-e7475536e65f.PNG)
Here's Chrome where the DOM and the actual code look the same structurally:
![slider-labelincontainer-chrome](https://user-images.githubusercontent.com/3506071/44811096-b7690e80-aba1-11e8-9d4b-59b6bd13f197.PNG)
